sh.sh

```diff
@@ -0,0 +1,32 @@
+read _ _ KERNEL _ </proc/version
+
+[ -z "$HOSTNAME" ] && read HOSTNAME </etc/hostname
+
+TASKS=0
+for dir in /proc/*; do
+  case $dir in
+  	*[0-9]*) TASKS=$((TASKS + 1)) ;;
```

Comment:
does this actually work, would it incorrectly match `x1` or `1x`? is it testing if a name contains only numbers, or contains any numbers?

Reply:
You are right, idk where I had my mind. I have found another reliable way to do it by testing whether comparing the process id with -1 fails